pom.xml

```diff
@@ -210,6 +210,11 @@
       <artifactId>support-log-formatter</artifactId>
       <version>1.2</version>
     </dependency>
+    <dependency>
+      <groupId>org.kohsuke</groupId>
+      <artifactId>access-modifier-suppressions</artifactId>
+      <version>${access-modifier-checker.version}</version>
+    </dependency>
```

Comment:
Can this new dependency enters in conflict with other deps from Jenkins?

Reply:
The version comes from Jenkins BOM, which is there to ensure that different dependencies that the plugin has are mutually compatible.

As for inter-plugin conflicts, the [docs](https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/) say that each plugin has a dedicated classloader at runtime, so I don't think there'll be any issues.